.circleci/config.yml

```diff
@@ -44,11 +44,13 @@ jobs:
       - run:
           name: Copy artifacts
           command: |
-            mkdir -p /tmp/feature_errors
-            docker cp cla-end-to-end:behave_local ../../../tmp/feature_errors
+            echo "Manually copying files because circleci docker executor does not support volumes"
```

Comment:
can you update the orb to do the same as well?

Reply:
Updated orb to copy artifacts into data folder now